src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -5636,7 +5637,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
             />
 
     <ItemGroup Condition="'$(PublishSingleFile)' == 'true'">
-      <PublishedSingleFileToBeCopied Include="$(PublishedSingleFilePath)" TargetPath="$(PublishedSingleFileName)"/>
+      <PublishedSingleFileToBeCopied Include="@(_DeploymentManifestFiles)"/>
```

Comment:
Would _DeploymentManifestFiles include application manifest or Launcher? Is there a scenario where we might fail to overwrite destination files?

Reply:
_DeploymentManifestFiles  would contain the application EXE and all other loose file dependencies that need to be copied (like coreclr, clrjit etc.). The application manifest and launcher and explicitly passed in the Copy task invokation below.

Reply:
There shouldn't be any failures for overwriting destination files (unless the file in in use). 

All ClickOnce copy task invocations uses the identical semantics including this particular copy. There are 2 flags that we honour: SkipUnchangedFiles (true by default) and OverwriteReadOnlyFiles (false by default). Otherwise, we will overwrite the destination file.

Reply:
OK - I was confused by "PublishSingleFile" property that conditions this code. Is this for single-file or both single and loose files scenarios?

Reply:
The SingleFile==true check applies to both FDD and SCD single-file scenarios. 

For FDD scenario, there are no loose runtime files to include so _FilesExcludedFromBundle will be empty and PublishedSingleFileToBeCopied will point to only the final EXE. 

SCD of a .NET 5.0 is the only known case where there are loose runtime files that need to be published alongside the final application EXE so PublishedSIngleFileToBeCopied will be a collection which contains the final EXE and the loose files that I mentioned in a previous reply.